### PR TITLE
Add EasyTether

### PR DESCRIPTION
**Pull request by szhu**
- Not sure if version is correct
- Not sure what license is correct

### CODE REVIEW

File: Casks/easytether.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'easytether' do
+  version 'beta'
+  sha256 '985cdbaaca59ab79ea91df9f0c0ee9b6292dbb3b3bd271f30b383fdff14b7d0d'
+
+  url 'http://www.mobile-stream.com/beta/easytether.dmg'
+  name 'EasyTether'
+  homepage 'http://www.mobile-stream.com/easytether/'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/easytether.rb]: `:commercial`.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

**Comment by szhu** [comment]
The URL has "beta" in it, so it'd be equivalent to perhaps "0.*" URL. In that case the correct version is still `:latest`?

**Comment by vitorgalvao** [comment]
Yes. What matters is the url is not versioned.

**Comment by szhu** [comment]
Got it, thank you! Fixed by 17bbf1607bc17bcb3bdf88d2bae9b4d8e96067ba.

**Comment by vitorgalvao** [comment]
You grabbed extra commits.

**Comment by szhu** [comment]
Wait I forgot to add `:commercial`, which is now fixed too. This PR should contain only one commit, 2315519ce7fda76fe35c4a5cf1272c210b136fe8, which should be correct.

**Comment by szhu** [comment]
Oh wait my bad, I see what you mean now.

**Comment by vitorgalvao** [comment]
Yeah, I meant extra (file) changes, not extra commits.

**Comment by szhu** [comment]
Okay sorry about that. d85d77af5c5c169aee69b7d878254a93f35204c3 is final.

**Pull request merged**